src/test/java/seedu/address/model/UniquePersonListTest.java

```diff
@@ -16,4 +26,130 @@ public void asObservableList_modifyList_throwsUnsupportedOperationException() {
         thrown.expect(UnsupportedOperationException.class);
         uniquePersonList.asObservableList().remove(0);
     }
+
+    //@@author charlesgoh
+    @Test
+    public void sortPerson_byName_bothOrders() {
```

Comment:
Another way to verify that the address book is correctly sorted would be to iterate through the actual list, then verifying that the entries are in ascending/descending order (through the use of a utility function like verifySorted(list, order)), but this should be ok as well.